plugin/manager.go

```diff
@@ -106,19 +101,7 @@ func (m *Manager) React(event *event.Event) error {
 	return nil
 }
 
-// handshakeConfig is used to just do a basic handshake between a plugin and host. If the handshake fails, a user
-// friendly error is shown. This prevents users from executing bad plugins or executing a plugin directory. It is a UX
-// feature, not a security feature.
-var handshakeConfig = goplugin.HandshakeConfig{
-	// The ProtocolVersion is the version that must match between EG core and EG plugins. This should be bumped whenever
-	// a change happens in one or the other that makes it so that they can't safely communicate.
-	ProtocolVersion: 1,
-	// The magic cookie values should NEVER be changed.
-	MagicCookieKey:   "EVENT_GATEWAY_MAGIC_COOKIE",
-	MagicCookieValue: "0329c93c-a64c-4eb5-bf72-63172430d433",
-}
-
 // pluginMap is the map of plugins we can dispense.
 var pluginMap = map[string]goplugin.Plugin{
-	"subscriber": &SubscriberPlugin{},
+	"reacter": &ReacterRPCPlugin{},
```

Comment:
Package-level map? I'm a bit wary of this but if it's not exported then I guess I can live with it...just need to ensure it's thread-safe when we need to fork any goroutines.